go/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/camunda/camunda-platform-get-started/go
 
-go 1.17
+go 1.20
```

Comment:
1.20 is the latest version, but I don't know if we wanted to bump the minimum pre-requisite.

Reply:
Sorry. I just noticed that. 
I would suggest to go for 1.19 as this is what we use in [zeebe go client](https://github.com/camunda/zeebe/blob/main/clients/go/go.mod#L3)

Reply:
```suggestion
go 1.19
```